docs/error-reporting/project-setup/storage-management.md

```diff
@@ -64,15 +64,31 @@ To query fingerprints:
 
 You can add additional filters or aggregations for `application.version` (or the attribute you use for the version of your game or app) to see if there are impacts on a particular version that require deletion.
 
-### Storage Sampling
+### Delete Group
+
+If you have a group of errors that have been identified as not useful or relevant, you can delete the group to manage storage. This action is available only for admins and groups with fewer than 100k errors.
+
+To delete a group:
+
+1. Go to the **Explore** view.
+1. From the **Group - fingerprint** column, select the **Group options**.
+1. Select **Delete Group**.
+1. From the **Delete selected group** dialog, select one of the following:
+   - **Delete all physical objects**: Allows you to retain metadata for aggregates and searching, but all the details of the errors will no longer be available for debugging purposes.
+   - **Delete physical objects and metadata**: Removes all data from your Backtrace instance.
+1. To confirm the action, enter **DELETE**, then select **Delete forever**.
+
+See the [Delete Errors by Query](https://www.loom.com/share/a0fc2d684d3b41e399d9efed67a817e9) video to learn how to query fingerprints and delete the group of errors.
```

Comment:
Per style guide, all videos should be linked from our SL YouTube. http://localhost:3000/contributing/style-guide/#videos
```suggestion
See the [Delete Errors by Query](https://www.loom.com/share/a0fc2d684d3b41e399d9efed67a817e9) video to learn how to query fingerprints and delete the group of errors.
```

Reply:
Done - I replaced the URL with the iframe that links to the SL Youtube account.